Mathlib/Algebra/Homology/Localization.lean

```diff
@@ -57,4 +62,87 @@ noncomputable def homologyFunctorFactors (i : ι) :
     Q ⋙ homologyFunctor C c i ≅ HomologicalComplex.homologyFunctor C c i :=
   Localization.fac _ (HomologicalComplex.homologyFunctor_inverts_qis C c i) Q
 
+variable {C c}
+
+lemma isIso_Q_map_iff_mem_qis {K L : HomologicalComplex C c} (f : K ⟶ L) :
+    IsIso (Q.map f) ↔ HomologicalComplex.qis C c f := by
+  constructor
+  · intro h
+    rw [HomologicalComplex.qis_iff, quasiIso_iff]
+    intro i
+    rw [quasiIsoAt_iff_isIso_homologyMap]
+    refine' (NatIso.isIso_map_iff (homologyFunctorFactors C c i) f).1 _
+    dsimp
+    infer_instance
+  · intro h
+    exact Localization.inverts Q (HomologicalComplex.qis C c) _ h
+
 end HomologicalComplexUpToQis
+
+end
+
+section
+
+variable (C : Type*) [Category C] {ι : Type*} (c : ComplexShape ι) [Preadditive C]
+  [CategoryWithHomology C] [(HomologicalComplex.qis C c).HasLocalization]
+
+lemma HomologicalComplexUpToQis.Q_inverts_homotopyEquivalences :
+    (HomologicalComplex.homotopyEquivalences C c).IsInvertedBy
+      HomologicalComplexUpToQis.Q :=
+  MorphismProperty.IsInvertedBy.of_subset _ _ _
+    (Localization.inverts Q (HomologicalComplex.qis C c))
+    (homotopyEquivalences_subset_qis C c)
+
+namespace HomotopyCategory
+
+/-- The class of quasi-isomorphisms in the homotopy category. -/
+def qis : MorphismProperty (HomotopyCategory C c) :=
```

Comment:
I would prefer
```suggestion
def quasiIso : MorphismProperty (HomotopyCategory C c) :=
```
Or do you think the length will quickly become annoying?

Reply:
Thanks for the suggestion. I do not think it will be annoying at all. I have made the change both for `HomologicalComplex.quasiIso` and `HomotopyCategory.quasiIso`.